### PR TITLE
Accept more content-types - [CLOUD-336]

### PR DESCRIPTION
**Pull request by Bellamyjared**
## Summary by CodeRabbit

- **New Features**
  - Expanded support for wake word training data uploads to include `audio/ogg` and `audio/mp4` formats.

- **Bug Fixes**
  - Improved handling and validation of URL parameters in wake word assistance functionality, ensuring better error responses for missing or incorrect parameters.

### CODE REVIEW

**Comment by Bellamyjared** [comment]
Do we want to add addition tests to check for new content-type?
Currently there is only a test to see if the content-type is bad, but there is no test to check for the correct content-types.

File: src/services/assist.ts [hunk 1]
```diff
@@ -4,7 +4,11 @@ import { WorkerEvent } from "../common";
 enum TRIGGER_PATH {
   WAKE_WORD_TRAINING_UPLOAD = "/assist/wake_word/training_data/upload",
 }
-const WAKE_WORD_ALLOWED_CONTENT_TYPES = ["audio/webm"];
+const WAKE_WORD_ALLOWED_CONTENT_TYPES = [
```
Comment (ludeeus) [inline src/services/assist.ts:7]: > Do we want to add addition tests to check for new content-type?
Currently there is only a test to see if the content-type is bad, but there is no test to check for the correct content-types.



The current good one (`"audio/webm"`) is tested as it's added to the base request.
But yes, adding a specific test for each valid entry would be good.

Comment (Bellamyjared) [reply src/services/assist.ts:7]: Sounds good, I'll create a new PR for those tests, if that's alright

**Comment by Bellamyjared** [comment]
Removing this commit and splitting PR more correctly

**Pull request closed without merging**